code/controllers/configuration.dm

```diff
@@ -446,7 +446,7 @@ GLOBAL_LIST_EMPTY(storyteller_cache)
 
 				if("ipr_minimum_age")
 					config.ipr_minimum_age = text2num(value)
-			
+
```

Comment:
It seems that this file wasn't actually changed, but the whitespace is being screwy. Harmless, but it'd probably be better if the changes to this were reverted somehow.